keps/sig-cloud-provider/2436-controller-manager-leader-migration/README.md

```diff
@@ -239,13 +240,13 @@ leaderName: cloud-provider-extraction-migration
 resourceLock: leases
 controllerLeaders:
   - name: route-controller
-    component: kube-controller-manager
+    component: *
   - name: service-controller
-    component: kube-controller-manager
+    component: *
   - name: cloud-node-controller
```

Comment:
Can we remove cloud-node-controller and cloud-nodelifecycle-controller as they are not part of the default. We should add in cloud-node-lifecycle as it is.

Reply:
I've replaced this part with the actual default configuration.